client/my-sites/invites/invite-accept-logged-in/index.jsx

```diff
@@ -34,7 +35,7 @@ export default React.createClass( {
 									usernameWrap: <span className="invite-accept-logged-in__join-as-username" />
 								},
 								args: {
-									username: user && user.display_name
+									username: user && decodeEntities( user.display_name )
```

Comment:
Maybe we should make this fix more generic? maybe around here: https://github.com/Automattic/wp-calypso/blob/master/server/user-bootstrap/shared-utils.js#L60